src/App.js

```diff
@@ -1,42 +1,82 @@
 import React, { useEffect, useState } from "react";
+import Nav from "./components/Layout/Nav";
+import "./styles/theme.css";
+import "./styles/main.css";
+import { Signup, ChangePassword, Login } from "./components/Auth/index";
+import Cart from "./components/Cart/Cart";
+import Wishlist from "./components/Wishlist/Wishlist";
+import ProductDetails from "./components/Products/ProductDetails/ProductDetails";
 import { Routes, Route } from "react-router-dom";
-import { fetchCategories } from "./services/index"
 import Home from "./components/Home/Home";
+import Profile from "./components/Profile/Profile";
+import ProductsListing from "./components/Products/ProductsListing";
+import { fetchAllProducts, initUserCart, fetchCategories, initUserAddress, initializeUserWishlist, updateCart } from "./services/index"
 import { useData } from "./context/data-context";
-import Nav from "./components/Layout/Nav";
-import "./styles/main.css";
+import { useAuthContext } from "./context/auth-context";
+import axios from "axios"
+import { Toast } from "./components/Toast/Toast";
+import { Order } from "./components/Private/Order/Order";
```

Comment:
You can create an index.js in components folder, do a named export and here you can import them using `./components` . No need of such long path names.